ping_one/README.md

```diff
@@ -1,42 +1,61 @@
-# Agent Check: ping_one
+# PingOne Integration for Datadog
```

Comment:
[Question] - Is it common to specify that our integration is for Datadog?

Reply:
No, We can remove it also.